docs/cloud/what-are-the-account-level-roles-for-users-in-temporal-cloud.md

```diff
@@ -0,0 +1,17 @@
+---
+id: what-are-the-account-level-roles-for-users-in-temporal-cloud
+title: What are the account-level Roles for users in Temporal Cloud?
+sidebar_label: Account-level Roles
+description: Account-level Roles are Global Admin, Developer, and Read-Only.
+---
+
+When a Global Admin invites a user to join an account, the Global Admin selects one of the following Roles for that user:
+
+- **Global Admin**
+  - Has full administrative permissions across the account, including users, usage, and billing.
```

Comment:
what does it mean to have both usage and billing permissions? I thought Cloud UI only supports usage?